vars/getJenkinsLabels.groovy

```diff
@@ -33,11 +35,29 @@ def call(String backend, String region=null) {
         if (label != null){
             println("Found AWS builder with label: " + label)
             return [ "label": label, "region": region ]
-        }
-        else{
+        } else {
             throw new Exception("=================== AWS region ${region} not supported ! ===================")
         }
 
+    }
+    else if (cloud_provider == 'gce' && datacenter)
```

Comment:
Amos, this is a copy-paste!!!
You could solve this by adding simple if and generalizing cloud provider-related strings.
```
    if cloud_provider == 'aws': {
        def supported_regions = ["eu-west-2", "eu-north-1", "eu-central-1"]
        }
    else if  cloud_provider == 'gce':
         def supported_regions = ["us-east1", "us-west1"]
         }
     else {
         throw new Exception("Unsupported cloud provider")
         }
```